syfertext/string_store.py

```diff
@@ -0,0 +1,54 @@
+from .utils import hash_string
+
+#############################
+# StringStore object acts as a lookup table
+# It looks up strings by hashes 
+#############################
+
+class StringStore:
+
+    def __init__(self, strings=None):
+        """Create the StringStore"""
+
+        self.key_to_string = {}
+        self.string_to_key = {}
+
```

Comment:
Please comment lines 13 and 14

Reply:
These dicts store the hashes and strings.